daffodil-lib/src/main/scala/org/apache/daffodil/schema/annotation/props/PropertyScoping.scala

```diff
@@ -131,6 +138,22 @@ trait FindPropertyMixin extends PropTypes {
           lr
         }
       }
+    /* if this looks like it could be an epression, issue a warning */
+    /* only if isForExpression is false*/
+    /* for runtime mixins, need to supply true, other places, no change*/
+    if( ! isForExpression ) {
+      propRes match{ 
+        case Found( v, _, _, _ ) => {
+
+          if( v.startsWith( "{")) {
+            SDW( WarnID.NonExpressionPropertyValueLooksLikeExpression, "Property %s that looks like an expression cannot be an expression: %s", pname, v )
```

Comment:
I'm definitely in favor of enabling scalariform. [DAFFODIL-2133](https://issues.apache.org/jira/browse/DAFFODIL-2133) is the issue for that. I'd like to enable the check on pull requests so it fails an PR's that don't follow the Daffodil style guidelines. The problem is I'm sure a lot of our code doesn't currently match the style guides, so we'd need to fix that first. Probably not a small task.

Reply:
Actually, enabling sclariform probably isn't a ton of work since it autoformats everything. I guess the real work is figuring out if any of the defaults need to be changed (hopefully not, hopefully we following the Scala style guides pretty closely), and then reviewing all of the changes. And hopefully a bunch of formatting changes wouldn't be too difficult to review.

Reply:
I've updated the formatting to be more in line with the rest of the file. I don't see any additional changes that need to be made in this context given the existence of DAFFODIL-2133